create-instance-profile.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+NAME=${1}
+
+# ReCreate a policy for codedeploy
+PDP="file://$(pwd)/codedeploy_policy_document.json"
+RDP="file://$(pwd)/codedeploy_role_document.json"
+
+POLICY=$(aws iam create-policy --policy-name ${NAME}_policy --policy-document ${PDP})
+POLICY_ARN=$(echo $POLICY | sed -e 's/^.*\"Arn\"/\"Arn\"/g' -e 's/\"Arn\": \"\([^\"]*\)\".*$/\1/g')
```

Comment:
Correct - updated to do so.

On 5 August 2015 at 17:18, Om Sharma <notifications@github.com> wrote:

> In create-instance-profile.sh
> <https://github.com/openregister/deployment/pull/8#discussion_r36321236>:
>
> > @@ -0,0 +1,23 @@
> > +#!/bin/bash
> > +
> > +NAME=${1}
> > +
> > +# ReCreate a policy for codedeploy
> > +PDP="file://$(pwd)/codedeploy_policy_document.json"
> > +RDP="file://$(pwd)/codedeploy_role_document.json"
> > +
> > +POLICY=$(aws iam create-policy --policy-name ${NAME}_policy --policy-document ${PDP})
> > +POLICY_ARN=$(echo $POLICY | sed -e 's/^.*\"Arn\"/\"Arn\"/g' -e 's/\"Arn\": \"\([^\"]*\)\".*$/\1/g')
>
> can use --query option to retrieve the required value from response
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openregister/deployment/pull/8/files#r36321236>.
>



-- 
Thanks,

/Saqib